src/v0/destinations/google_adwords_offline_conversions/transform.js

```diff
@@ -41,22 +44,32 @@ const getConversions = (message, metadata, { Config }, event, conversionType) =>
   const { properties, timestamp, originalTimestamp } = message;
 
   const filteredCustomerId = removeHyphens(customerId);
+  const eventLevelConsentsData = getConsentsDataFromIntegrationObj(message, conversionType);
+
   if (conversionType === 'click') {
     // click conversion
     const convertedPayload = getClickConversionPayloadAndEndpoint(
       message,
       Config,
       filteredCustomerId,
+      eventLevelConsentsData,
     );
     payload = convertedPayload.payload;
     endpoint = convertedPayload.endpoint;
   } else if (conversionType === 'store') {
-    payload = getStoreConversionPayload(message, Config, filteredCustomerId);
+    payload = getStoreConversionPayload(
+      message,
+      Config,
+      filteredCustomerId,
+      eventLevelConsentsData,
```

Comment:
For store conversions, why eventLevelConsentData is being passed ?

Reply:
removed it , hope it is not looking confusing any more

Reply:
Not only confusion, when an attribute is not being used, why to pass